stock_move_actual_date/models/actual_date_mixin.py

```diff
@@ -16,10 +16,12 @@ class ActualDateMixin(models.AbstractModel):
         compute="_compute_is_editable_actual_date", string="Is Editable"
     )
 
+    def _allow_editable(self):
```

Comment:
Let's make the method name more self-explanatory.
```suggestion
    def _check_actual_date_editable(self):
```

Reply:
@yostashiro 
I'm not satisfied with the current design as it allows editing in unintended models (e.g., `mrp.production`) if they inherit this `mixin` model. `actual_date` in mrp.production should only be editable by an `MRP Manager` or when the state is not `canceled` or `done`. Currently, a user with both` Stock Manager` and` MRP User` roles can still edit `actual_date`, and the same issue exists in `stock.picking`.

Instead of adding user group checks in the `mixin` model, we should inherit the method in each related model and apply specific group checks. However, this approach introduces redundancy (e.g., checking Stock Manager in both `stock.picking` and `stock.scrap` for the same logic). 
Do you have any idea on this?

Reply:
As we discussed, I added the new group and applied the changes.